src/test/java/seedu/doctorappointments/DoctorAppointmentCheckerTest.java

```diff
@@ -0,0 +1,62 @@
+package seedu.doctorappointments;
+
+import org.junit.jupiter.api.Test;
+import seedu.model.doctorappointment.AppointmentList;
+import seedu.logic.errorchecker.DoctorAppointmentChecker;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Generally it would be better to not use blanket imports. Perhaps you could import only the stuff you need for Java CI?